Marlin/Configuration.h

```diff
@@ -981,16 +981,16 @@
   #define X_MIN_ENDSTOP_INVERTING true  // Set to true to invert the logic of the endstop.
   #define Y_MIN_ENDSTOP_INVERTING true  // Set to true to invert the logic of the endstop.
   #if BOTH(KNUTWURST_BLTOUCH, KNUTWURST_4MAXP2)
-      #define Z_MIN_ENDSTOP_INVERTING false  // Set to true to invert the logic of the endstop.
-  #else
       #define Z_MIN_ENDSTOP_INVERTING true  // Set to true to invert the logic of the endstop.
+  #else
+      #define Z_MIN_ENDSTOP_INVERTING false  // Set to true to invert the logic of the endstop.
```

Comment:
This only applies of 4MAX Pro v1, or am I wrong? It would break the v2.0 config.

Reply:
Yes only 4MaxPro1